monitord.conf

```diff
@@ -34,7 +34,9 @@ sshd.service
 
 [machines]
 enabled = true
-auto_discover = true
```

Comment:
So we've made the decision we're only doing auto discovery?

Reply:
It doesn't really give us anything valuable. We can achieve exactly the same effect by not defining allowlist nor blocklist. If we define one of those though, we limit the resources to selected group. This is a matter of approach, this one mimics the logic from `units`.